lib/motions/general-motions.coffee

```diff
@@ -471,14 +471,20 @@ class ScrollKeepingCursor extends Motion
 
   execute: (count) ->
     # TODO: remove this conditional once after Atom v1.1.0 is released.
-    if @editor.setFirstVisibleScreenRow?
-      newTopRow = @getNewFirstVisibleScreenRow(count)
-      super(count)
-      @editor.setFirstVisibleScreenRow(newTopRow)
-    else
-      scrollTop = @getNewScrollTop(count)
-      super(count)
-      @editorElement.setScrollTop(scrollTop)
+    # TODO: Atom v1.1.0 was released.
+    # But setFirstVisibleScreenRow() use getLineCount() not
+    # getScreenLineCount(). Once after it use getScreenLineCount(),
+    # [1. by setFirstVisibleScreenRow] can be changed to [2. by scrollTop]
```

Comment:
> But setFirstVisibleScreenRow() use getLineCount() not getScreenLineCount()

Thanks so much for catching this! I think that we should just fix that method. There are some issues with using the `TextEditorElement::getScrollTop`, because it doesn't always update until the next animation frame.